src/main/java/gov/usds/case_issues/model/NoteSummary.java

```diff
@@ -51,6 +59,10 @@ public String getUserId() {
 		return userId;
 	}
 
+	public String getUserName() {
```

Comment:
Courtesy of my late realization that I was saying I wanted to talk about the API as well: I think a flat API object is the wrong thing, here—we probably want to be able to embed more information about the created-by user, and that's going to be a recurring thing, so we should embed some kind of user object as "createdBy", or even copy the timestamp into a "created" object that's actually a generalized audit object with a user and a timestamp. Not sure which, but it seems worth talking about for a few minutes.